### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Carreau**
Try to explain a bit what is this in readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+Hello IPythonista, 
+
+You have a small extension for IPython, and you want it to be more visible ?
+You can request it to be hosted here. We will be happy to count you in the organisation. 
+
+Our goal is to make small extension easy to find, so for 1 line extension we created this repo. 
+That make it easier to maintain and respod to issues that going through N repository hosted by M user
+and where you never remember where is what. 
+
+If you have a bigger extension and what it on it's own repo, it's ok too,
```
Comment (jankatins) [inline README.md]: s/what/want/?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+we can create teh repo  and give you admin right. 
+
+One thing we want to avoid is project being abandoned by their author without a way
+to update the readme, or give contribution access to people interested. 
+
+So if you contribute an extension there, and just go away we won't mind, and if a extension is
+abandonned and want to adopt it, the res of the organisation can give you cwrite access to update it.
```
Comment (jankatins) [inline README.md]: s/cwrite/write/

File: README.md [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+we can create teh repo  and give you admin right. 
+
+One thing we want to avoid is project being abandoned by their author without a way
+to update the readme, or give contribution access to people interested. 
+
+So if you contribute an extension there, and just go away we won't mind, and if a extension is
+abandonned and want to adopt it, the res of the organisation can give you cwrite access to update it. 
+
+This is roughly base on how https://github.com/ipython-contrib/IPython-notebook-extensions works
+and we shoudl be more explicit there too.
```
Comment (jankatins) [inline README.md]: s/shoudl/should/

**Comment by jankatins** [comment]
I like your text more than my try in https://github.com/ipython-contrib/IPython-extensions/pull/5, but in #5, there are also some bits about installing/...

Would it be possible to submit this PR as a PR against #5?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,29 @@
 # IPython-extensions
-Extensions for IPython
+
+Hello IPythonista, 
+
+You have a small extension for IPython, and you want it to be more visible ?
+You can request it to be hosted here. We will be happy to count you in the organisation. 
+
+Our goal is to make small extension easy to find, so for 1 line extension we created this repo. 
+That make it easier to maintain and respod to issues that going through N repository hosted by M user
```
Comment (jankatins) [inline README.md]: s/respod/respond/

**Comment by Carreau** [comment]
AH, I missed #5, I'll merge it and rebase mine on top.

**Comment by jankatins** [comment]
@Carreau Note that #5 mentions installign via pip, which isn't possible yet, as there is no release...

**Comment by Carreau** [comment]
> @Carreau Note that #5 mentions installign via pip, which isn't possible yet, as there is no release...

It's ok, we can do that soon.

**Comment by Carreau** [comment]
Also I think we shoudl publish under `ipyext`  to have the same package name and import.

**Comment by jankatins** [comment]
@Carreau: rename to `ipyext` sounds reasonable: https://github.com/ipython-contrib/IPython-extensions/pull/8

**Pull request closed without merging**